handbook/engineering/observability/monitoring_pillars.md

```diff
@@ -119,44 +122,21 @@ The point is _you have to give a best-guess at what a bad or good value looks li
 
 See also ["the difference between Warning and Critical alerts"](https://docs.sourcegraph.com/admin/observability/alerting_custom_consumption#the-difference-between-critical-and-warning-alerts) for how we communicate this to site admins.
 
-#### What you should do instead
+#### What you should do
 
 Declare a `Warning` alert, and give it your best guess at what a threshold would look like. It's OK if it's flaky, or fires when there isn't a problem sometimes or on some Sourcegraph instances.
 
 Over time, we will adjust the alert so it becomes more reliable and becomes the baseline number that most Sourcegraph instances observe. Once it becomes really reliable (and empirically proven to be so, based on pings we get from customer instances), it can be promoted to a `Critical` alert as well if it speaks of a critical part of Sourcegraph.
 
-### FAQ: Why can't I create a dashboard in the WYSIWYG Grafana editor?
-
-This violates the third pillar of monitoring at Sourcegraph:
-
-> Creating dashboards outside of the monitoring generator, such as with the Grafana WYSIWYG editor, is forbidden. All metrics should be presented in the most plain, mundane, non-fanciful way that every site admin can understand.
-
-#### Why this is bad
+#### Exceptions
 
-Using the Grafana WYSIWYG editor, or otherwise operating outside the constraints of [the monitoring generator](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/tree/monitoring) makes it:
+TODO
```

Comment:
@tsenart TODO what graphs are good to _not_ include alerts on

Reply:
There are no specific graphs. Building a dashboard is an iterative endeavour, over the course of many deployments, observation of graphs to learn what good and bad is, and subsequent tweaking, and *possibly* definition of alerts, when useful / applicable.

Reply:
I think @arussellsaw's description here https://github.com/sourcegraph/about/pull/2000#discussion_r523034778 provides a step towards a concrete exception :) Some guideline for "informational vs actionable" would be good to have here!

Reply:
to be clear my personal take is that engineers should be as free as possible to make whichever informational dashboards as they want, but actionable ones should absolutely meet certain standards

Reply:
> should be as free as possible to make whichever informational dashboards as they want

I think we should properly support informational dashboards, but this particular snippet I thought is a good argument for having _some_ recommendation towards one or the other: https://github.com/sourcegraph/about/pull/2000/files#diff-c74c0b112f5eaedf8f54f56b54ebaff1294645e9d10106829776938eaec60d0aR186

> [...] it's easier for us to say:
>
>> Go to the **Searcher** dashboard, scroll down and expand the **Internal service requests** panel and send us a screenshot.
>
> Than it is for us to say:
>
>> On the **Searcher** dashboard, send us a screenshot of the whole page (I know it's really long, sorry about that, if you could just make sure to include the "Frontend request failures over 5m" panel and the [...] panels, that is all we should need)

Reply:
Though now that I think of it, maybe it's not really that important and teams should just make sure their dashboards are of reasonable size themselves :)

Reply:
Resolving, since I just removed the pillar: https://github.com/sourcegraph/about/pull/2000#discussion_r523886716